src/EntryPoint/RestEntryPoints.ts

```diff
@@ -51,11 +52,36 @@ export class RestEntryPoints implements EntryPoints {
     });
   }
 
-  public async entrypoints(projectId: string): Promise<EntryPoint[]> {
-    const res = await this.client.get(
-      `/api/v2/projects/${projectId}/entry-points`
-    );
+  public async entrypoints(
+    projectId: string,
+    limit: number = 10
+  ): Promise<EntryPoint[]> {
+    let remaining = limit;
+    const data: EntryPoint[] = [];
+    let nextId: string;
+    let nextCreatedAt: string;
 
-    return res.data.items;
+    while (remaining > 0) {
+      const {
+        data: { items = [] }
+      } = await this.client.get(`/api/v2/projects/${projectId}/entry-points`, {
+        params: {
+          nextId,
+          nextCreatedAt,
+          limit: Math.min(remaining, this.entrypointsPaginationBatchSize)
+        }
+      });
+
+      if (!items || items.length === 0) {
```

Comment:
You can simplify the condition (items will never accept undefined or null, see L66)
```suggestion
      if (!items.length) {
```